### PR TITLE
Feature/calculator UI

### PR DESCRIPTION
**Pull request by ShinhyoBelok**
# Changes ♻️ 

In this pull request, I develop a React component that will hold the core functionality of a calculator.

* Delete all the boilerplate from CRA (text, images, styles).
* Create a React component Calculator.js
* Add CSS to Calculator.js

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          <button type="button">÷</button>
+          <button type="button">x</button>
+          <button type="button">-</button>
+          <button type="button">+</button>
+          <button type="button">=</button>
+        </div>
+      </div>
+    </div>
+  </main>
+);
```
Comment (sja-thedude) [inline src/components/Calculator.js]: - [ ] Kindly make sure that `Calculator.js` component should be class-based.

Example:

```
class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}
```

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,8 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
   );
 }
```
Comment (sja-thedude) [inline src/App.js]: - [ ] Kindly make sure that `App.js` component should also be class-based.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ShinhyoBelok, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Correct Gitflow
✔️ Professional Readme

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 3]
```diff
@@ -1,29 +1,12 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React from 'react';
+import Calculator from './components/Calculator';
+// eslint-disable-next-line
```
Comment (ShadyShawkat) [inline src/App.js]: - [ ] Please remove the `eslint-disable` line and try to fix it manually as this defies the purpose of using linters in the first place.

File: src/components/Calculator.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './Calculator.css';
+// eslint-disable-next-line
```
Comment (ShadyShawkat) [inline src/components/Calculator.js]: - [ ] Please remove the `eslint-disable` line and try to fix it manually as this defies the purpose of using linters in the first place.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @ShinhyoBelok, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 4 above)
Comment (ShinhyoBelok) [reply src/components/Calculator.js:3]: Yes, I understand, but the linters are forcing me to change the class base component to pure function, and when I submitted it as function the reviewer told me to change it to class base component. Here is the linter error:
image

**Review by julie-ify — APPROVED** [review]
Hi @ShinhyoBelok,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**